### ISSUE TITLE
Windows Bitmap Image with 8-bit index color (V4 header) has a strange color

### ISSUE THREAD

**Issue by kntroh** [wxWidgets/Phoenix#1580]
**Operating system**: Windows 10 Pro 64-bit
**wxPython version & source**:  4.0.7.post2 from pypi
**Python version & source**:   Python 3.8.0
**Description of the problem**: Windows Bitmap Image with 8-bit index color (V4 header) has a strange color.

A 8-bit index color bitmap image (V4 header) created by GIMP 2.10.18 (and probably earlier) is showing strange colors.

![indexcolor_v4](https://user-images.githubusercontent.com/1318355/77530586-eefe8080-6ed4-11ea-84c0-2bfb81360c91.png) -> ![indexcolor_v4_shown](https://user-images.githubusercontent.com/1318355/77530659-09d0f500-6ed5-11ea-9039-4c400d6a137f.png)

```python
import wx
app = wx.App()
frame = wx.Frame(None, -1, 'Index Color V4',size=(150,150))
bmp = wx.StaticBitmap(frame, -1)
bmp.SetBitmap(wx.Bitmap('indexcolor_v4.bmp'))
frame.Show()
app.MainLoop()
```

Attaching the file I made for the test: [indexcolor_v4.zip](https://github.com/wxWidgets/Phoenix/files/4380449/indexcolor_v4.zip)

Steps to Create:
 1.  Create a color image and open it in the GIMP.
 2. `Menu > Image > Mode > Indexed...`
 3. Select `Colormap > Generate optimum palette` and push Convert.
 4. `File > Export As...`
 5. Export image with extension `.bmp`.

**Comment by RobinD42** [comment]
This would be a wxWidgets issue, not wxPython specific. Please create a ticket for this at https://trac.wxwidgets.org/, if one doesn't already exist, and add a link to the ticket here so we can track it.

**Comment by kntroh** [comment]
https://trac.wxwidgets.org/ticket/18634
I did search in the Trac and this problem had already been found and fixed. It will probably be released in wxWidgets 3.1.4, so I'll wait it. Thanks.
